scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -615,12 +615,11 @@ def summary(self, results, ignore_unrecognized_sections, duration):
                 f'{", " + str(results.none_cases) + " without a status" if results.none_cases else ""}'
                 f' on {len(self.filtered_platforms)} out of total {total_platforms} platforms ({platform_rate:02.2f}%).'
             )
-            if results.skipped_cases or results.filtered_cases or results.notrun_cases:
+            if results.skipped_cases or results.notrun_cases:
                 logger.info(
-                    f'{results.skipped_cases + results.filtered_cases} selected test cases not executed:' \
+                    f'{results.skipped_cases + results.notrun_cases} selected test cases not executed:' \
```

Comment:
with `./scripts/twister -p sam_e70_xplained/same70q21 -T tests/subsys/dsp/basicmath`
this PR:
```
Total test cases: 949
├─ Filtered test cases:   1
└─ Executed test cases: 948
   ├─ Passed test cases:         0
   ├─ Built only test cases:   948
INFO    - 948 selected test cases not executed: 948 not run (built only).
```
'selected not executed` built-only under its 'Executed` root looks strange, mb it should be
`└─ Selected test cases: 948` ?

before it was also strange, but differently )
```
INFO    - 1 selected test cases not executed: 1 filtered, 948 not run (built only).
```

Reply:
not sure what the problem is? This all looks correct.

Reply:
oh, you mean this:
```
└─ Executed test cases: 948
   ├─ Passed test cases:         0
   ├─ Built only test cases:   948
```

Reply:
yeah, this looks better.